plugins/scaffolder/src/translation.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTranslationRef } from '@backstage/core-plugin-api/alpha';
+
+/** @alpha */
+export const scaffolderTranslationRef = createTranslationRef({
+  id: 'scaffolder',
+  messages: {
+    activity_for_task: 'Activity for task: {{taskId}}',
```

Comment:
I sorted the keys, but the [eslint sort-keys rule](https://eslint.org/docs/latest/rules/sort-keys) don't have a fixable version (for a good reason https://github.com/eslint/eslint/issues/7714). I used this [extesion](https://marketplace.visualstudio.com/items?itemName=zengxingxin.sort-js-object-keys) for this, but maybe we can code a simple sort for our use case.